dashboard/src/components/schedule/ScheduleView.vue

```
@@ -1,30 +1,54 @@
-    class="flex gap-4 w-full my-6"
-    :class="{
-      'flex-col': view === 'vertical',
-      'overflow-x-scroll flex-row min-h-[800px]': view === 'horizontal',
-    }"
-  >
+  <div class="w-full my-6 relative">
+    <!-- Fake scrollbar container at the top -->
     <div
-      v-for="(sessions, hall) in schedule"
-      :key="hall"
+	  v-if="view === 'horizontal'"
```

ESLint

[error] Replace `↹` with `····`

(prettier/prettier)